pom.xml

```diff
@@ -235,7 +235,7 @@
 			<dependency>
 				<groupId>org.hibernate</groupId>
 				<artifactId>hibernate-validator</artifactId>
-				<version>5.4.3.Final</version>
+				<version>7.0.1.Final</version>
```

Comment:
Is this the latest version of hibernate validator ?

Reply:
I have tried almost all the versions with correct group Id, dosent work for me